packages/block-editor/src/components/justify-toolbar/style.scss

```diff
@@ -0,0 +1,16 @@
+// Justification.
```

Comment:
This is really specific to `Navigation` block structure, which applies the `.items-justified-left || right...etc` css classes to a parent container, which has a child `ul`. Most blocks I think apply the class to the parent element and have afterwards the children to be positioned.

This is tricky here as it will not work well for everything. I think we should cover the more generic case and change accordingly the individual blocks that have the less common case.

For example buttons will need just: `.items-justified-left { justify-content: flex-start }`

Reply:
Addressed in discussion below.